optaplanner-examples/src/main/java/org/optaplanner/examples/nqueens/app/NQueensApp.java

```diff
@@ -91,7 +91,7 @@ public NQueensApp() {
 
         ScoreDirectorFactoryConfig scoreDirectorFactoryConfig = new ScoreDirectorFactoryConfig();
         scoreDirectorFactoryConfig.setScoreDrlList(
-                Arrays.asList("org/optaplanner/examples/nqueens/solver/nQueensConstraints.drl"));
+                Arrays.asList("org/optaplanner/examples/nqueens/optional/score/drools/nQueensConstraints.drl"));
```

Comment:
(soft)

Given that CS is the default (and not in the optional package),
FooEasyJavaScoreCalculator is in the package o.o.examples.foo.optional.score
FooIncrementalJavaScoreCalculator is in the package o.o.examples.foo.optional.score
So, I'd argue that
fooConstarints.drl should be in the package o.o.examples.foo.optional.score too,
not o.o.examples.foo.optional.score.drools.
=> (drop drools subpackage). Wdyt?

Reply:
Sure, no problem.

Reply:
Thanks :)